content/management/configuration-api/v3.2/index.mdx

```diff
@@ -32,6 +32,720 @@ Summing up, if the user token starts with `fra-`, you should add the `X-Region:
+
+#### Specifics
+
+|                 |                                                                   |
+|-----------------|-------------------------------------------------------------------|
+| Method URL      | `https://api.livechatinc.com/v3.2/configuration/action/get_agent` |
+| Required scopes | `agents--my:ro` `agents--all:ro`                                  |
+
+#### Request
+
+| Request object        | Type       | Required | Notes                                |
+|-----------------------|------------|----------|--------------------------------------|
+| `id`                  | `string`   | Yes      | Agent ID                             |
+| `fields`__*__         | `string[]` | No       | Additional fields to include.        |
+
+__*__ The table below presents the possible values for the `fields[]` parameter:
+
+| Possible value        | Notes                                                       |
+|-----------------------|-------------------------------------------------------------|
+| `work_scheduler`      | Work scheduler object for agent                             |
+| `groups`              | Groups in which agent belongs                               |
+| `email_subscriptions` | Active agent email subscriptions list                       |
+| `notifications`       | Enabled agent email subscriptions list                      |
+| `job_title`           | Agent's job title                                           |
+| `mobile`              | Agent's mobile number                                       |
+| `max_chats_count`     | Agent's maximum number of concurrent chats                  |
+| `suspended`           | Boolean value  indicating if agent is suspended             |
+| `awaiting_approval`   | Boolean value  indicating if agent is awaiting for approval |
+
+
+</Text>
+
+<Code>
+
+<CodeSample path={'REQUEST'}>
+
+```shell
+curl -X POST \
+  https://api.livechatinc.com/v3.2/configuration/action/get_agent \
+  -H 'Authorization: Bearer <your_access_token>' \
+  -H 'Content-Type: application/json' \
+  -d '{
+        "id": "agent@domain.com",
+        "fields": [
+          "summaries",
+          "max_chats_count",
+          "job_title"
+        ]
+      }'
+```
+
+</CodeSample>
+
+<CodeResponse>
+
+```json
+{
+    "id": "agent@domain.com",
+    "name": "John",
+    "avatar_path": "https://domain.com/avatar.image.jpg",
+    "role": "administrator",
+    "login_status": "accepting chats",
+    "job_title": "Support Hero",
+    "max_chats_count": 5,
+    "summaries": [
+      "daily_summary",
+      "weekly_summary"
+    ]
+}
+```
+
+</CodeResponse>
+
+</Code>
+
+</Section>
+
+<Section>
+
+<Text>
+
+### List Agents
+
+Returns all Agents within a license. You can request only the Agents from a specific group. Some of agents fields are returned by default by you can also extend them via providing fields names.
```

Comment:
```suggestion
Returns all Agents within a license. You can request only the Agents from a specific group. 
```

Reply:
same as above. it's fine as long as we specify which fields are required/optional in the table